### PR TITLE
Update documentation regarding PET tracers

### PR DESCRIPTION
**Pull request by alexandreroutier**
Hi!

I found some reminding old convention names in Clinica documentation. They have been adressed in this PR 🤞 

Alexandre

### CODE REVIEW

File: clinica/pipelines/pet_surface/pet_surface_longitudinal_cli.py [hunk 1]
```diff
@@ -31,14 +31,17 @@ def cli(
 ) -> None:
     """Longitudinal surface-based processing of PET images.
 
-       ACQ_LABEL corresponds to the label given to the PET acquisition, specifying the tracer used. Frequently used values are 'fdg'
-     or 'av45'.
+       ACQ_LABEL corresponds to the label given to the PET acquisition, specifying the tracer used.
+    Frequently used values are 'FDG' or '18FAV45'.
```
Comment (ghisvail) [inline clinica/pipelines/pet_surface/pet_surface_longitudinal_cli.py:35]:
```suggestion
    Frequently used values are '18FFDG' or '18FAV45'.
```

File: clinica/pipelines/pet_surface/pet_surface_longitudinal_cli.py [hunk 2]
```diff
@@ -31,14 +31,17 @@ def cli(
...
 
-       The reference region must be precised to perform intensity normalization. Accepted values include: 'pons', 'cerebellumPons', 'pons2', 'cerebellumPons2'.
+       The reference region must be precised to perform intensity normalization.
+    Accepted values include: 'pons', 'cerebellumPons', 'pons2', 'cerebellumPons2'.
 
        PVC_PSF_TSV is the TSV file containing the psf_x, psf_y and psf_z of the PSF for each PET image.
 
-    https://aramislab.paris.inria.fr/clinica/docs/public/latest/Pipelines/PET_Surface_Longitudinal/
+    Prerequisite: You need to have performed the t1-freesurfer-longitudinal pipeline on your T1-weighted MR images.
```
Comment (ghisvail) [inline clinica/pipelines/pet_surface/pet_surface_longitudinal_cli.py:42]: :+1:

File: clinica/pipelines/pet_volume/pet_volume_cli.py [hunk 3]
```diff
@@ -71,12 +71,14 @@ def cli(
...
     """SPM-based pre-processing of PET images.
 
-       GROUP_LABEL is an user-defined identifier to target a specific group of subjects. For this pipeline, it is associated to the DARTEL template that you had created when running the t1-volume pipeline.
+       GROUP_LABEL is an user-defined identifier to target a specific group of subjects.
+    For this pipeline, it is associated to the DARTEL template that you had created when running the t1-volume pipeline.
 
-       ACQ_LABEL corresponds to the label given to the PET acquisition, specifying the tracer used. Frequently used values are 'fdg'
-     or 'av45'.
+       ACQ_LABEL corresponds to the label given to the PET acquisition, specifying the tracer used.
+    Frequently used values are '18FDG' or '18FAV45'.
```
Comment (ghisvail) [inline clinica/pipelines/pet_volume/pet_volume_cli.py:78]:
```suggestion
    Frequently used values are '18FFDG' or '18FAV45'.
```

File: docs/BIDS.md [hunk 4]
```diff
@@ -52,11 +52,17 @@ BIDS_Dataset/
...
-    in BIDS (pib -> 11CPIB, av45 -> 18FAV45, fbb -> 18FFBB, fdg -> 18FFDG, flute -> 18FFMM, tau -> 18FAV1451).
+    Since version 0.6, Clinica is now compliant with the official specifications for PET modality.
+    This involves three main changes:
+
+    - `task-rest` key/entity is now optional and ignored by Clinica
+    - PET acquisitions with different tracers now use the new tracer entity `trc` instead of
+    BIDS acquisition entity `acq`.
+    - PET files must use tracer entity `trc` (optional in BIDS but necessary for Clinica pipelines) and
+    label names follow the proposed convention in BIDS
+    (pib -> 11CPIB, av45 -> 18FAV45, fbb -> 18FFBB, fdg -> 18FFDG, flute -> 18FFMM, tau -> 18FAV1451).
```
Comment (ghisvail) [inline docs/BIDS.md:65]: :+1:

File: clinica/pipelines/pet_linear/pet_linear_cli.py [hunk 5]
```diff
@@ -41,9 +41,11 @@ def cli(
 ) -> None:
     """Affine registration of PET images to the MNI standard space.
 
-       ACQ_LABEL corresponds the label given to the PET acquisition, specifying the tracer used. Frequently used values are 'fdg' or 'av45'.
+       ACQ_LABEL corresponds the label given to the PET acquisition, specifying the tracer used.
+    Frequently used values are '18FFDG' or '18FAV45'.
 
-       The reference region must be precised to perform intensity normalization. Accepted values include: 'pons', 'cerebellumPons', 'pons2', 'cerebellumPons2'.
+       The reference region must be precised to perform intensity normalization.
```
Comment (omar-rifai) [inline clinica/pipelines/pet_linear/pet_linear_cli.py:47]:
```suggestion
       The reference region must be specified to perform intensity normalization.
```

File: clinica/pipelines/pet_surface/pet_surface_cli.py [hunk 6]
```diff
@@ -31,13 +31,16 @@ def cli(
 ) -> None:
     """Surface-based processing of PET images.
 
-       ACQ_LABEL corresponds to the label given to the PET acquisition, specifying the tracer used. Frequently used values are 'fdg'
-     or 'av45'.
+       ACQ_LABEL corresponds to the label given to the PET acquisition, specifying the tracer used.
+    Frequently used values are '18FFDG' or '18FAV45'.
 
-       The reference region must be precised to perform intensity normalization. Accepted values include: 'pons', 'cerebellumPons', 'pons2', 'cerebellumPons2'.
+       The reference region must be precised to perform intensity normalization.
```
Comment (omar-rifai) [inline clinica/pipelines/pet_surface/pet_surface_cli.py:37]:
```suggestion
       The reference region must be specified to perform intensity normalization.
```

**Review by ghisvail — APPROVED** [review]

File: clinica/pipelines/pet_volume/pet_volume_cli.py [hunk 7]
```diff
@@ -71,12 +71,14 @@ def cli(
...
 
-       The reference region must be precised to perform intensity normalization. Accepted values include: 'pons', 'cerebellumPons', 'pons2', 'cerebellumPons2'.
+       The reference region must be precised to perform intensity normalization.
```
Comment (omar-rifai) [inline clinica/pipelines/pet_volume/pet_volume_cli.py:80]:
```suggestion
       The reference region must be specified to perform intensity normalization.
```

**Comment by ghisvail** [comment]
As always, thanks @alexandreroutier for cleaning up behind me :+1: 

I liked the prerequisites section you have added.

I have corrected a few typos in the process.

**Review by omar-rifai — APPROVED** [review]
Thanks @alexandreroutier, just a few typos I found from previous  edits.

**Comment by ghisvail** [comment]
It reminds me, what should we do about `--acq-label` option? I believe it should become `--trc-label` at some point, in case our pipelines happen to process a dataset using an `acq` entity. What do you think @alexandreroutier @omar-rifai ?

We could also delay that change until we have reached full compliance after the coming sprints. I am open to both really.

**Comment by alexandreroutier** [comment]
> It reminds me, what should we do about `--acq-label` option? I believe it should become `--trc-label` at some point, in case our pipelines happen to process a dataset using an `acq` entity. What do you think @alexandreroutier @omar-rifai ?
> 
> We could also delay that change until we have reached full compliance after the coming sprints. I am open to both really.

Hi!

That is an issue I forgot to send yesterday, this is now in issue #637. I propose to discuss in this seperated issue. For me, this would mainly concern CLIs + TSV file for PVC computation.

**Comment by omar-rifai** [comment]
> > It reminds me, what should we do about `--acq-label` option? I believe it should become `--trc-label` at some point, in case our pipelines happen to process a dataset using an `acq` entity. What do you think @alexandreroutier @omar-rifai ?
> > We could also delay that change until we have reached full compliance after the coming sprints. I am open to both really.
> 
> Hi!
> 
> That is an issue I forgot to send yesterday, this is now in issue #637. I propose to discuss in this seperated issue. For me, this would mainly concern CLIs + TSV file for PVC computation.

Agreed, let's keep it for now (for release 0.6) it's not absurd and does not impact our compliance.

**Pull request merged**